src/Storage/Storage.Management/help/Get-AzStorageAccountManagementPolicy.md

```diff
@@ -0,0 +1,198 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.Management.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/en-us/powershell/module/Az.storage/get-Azstorageaccountmanagementpolicy
+schema: 2.0.0
+---
+
+# Get-AzStorageAccountManagementPolicy
+
+## SYNOPSIS
+Gets the management policy of an Azure Storage account.
+
+## SYNTAX
+
+### AccountName (Default)
+```
+Get-AzStorageAccountManagementPolicy [-ResourceGroupName] <String> [-StorageAccountName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### AccountResourceId
+```
+Get-AzStorageAccountManagementPolicy [-StorageAccountResourceId] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### AccountObject
+```
+Get-AzStorageAccountManagementPolicy -StorageAccount <PSStorageAccount>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzStorageAccountManagementPolicy** cmdlet gets the management policy of an Azure Storage account.
+
+## EXAMPLES
+
+### Example 1: Get the management policy of a Storage account.
+```
+PS C:\>Get-AzStorageAccountManagementPolicy -ResourceGroupName "myresourcegroup" -AccountName "mystorageaccount"
+
+ResourceGroupName  : myresourcegroup
```

Comment:
By default, the output should be table formatted.  You should show the table format output